spec/requests/account_spec.rb

```diff
@@ -15,7 +15,7 @@
   let(:userinfo) { {} }
 
   before do
-    log_in(user.email, user.password)
+    sign_in user
```

Comment:
I was tracking all the other changes here,
- the describe headers all make sense
- the vist / get makes sense (more explicit?  looks good)
- using resonse.body and have_content, great!


But not so sure about thsi one, what happened to log_in?
Did we define a new `sign_in` method somewhere?

Reply:
`log_in` is a helper method we wrote, it works for feature specs.  `sign_in` is a helper provided by Devise, it works for request specs.